src/psykal/mydomain_kern.f90

```diff
@@ -1,16 +1,28 @@
 module mydomain_kern_mod
 !
+use kind_params_mod, only : wp
+!
 implicit none
 !
+!type, extends(kernel_type) :: mydomain_type
+!  type(arg), dimension(2) :: meta_args = (/ &
+!    arg(WRITE, DIMS(3), CT), &
+!    arg(READ,  DIMS(3), CU, STENCIL(U)) /)
```

Comment:
Yes, I thought I'd use up and down as it made sense to me (even though I got them the wrong way round!). If you or Silvia know of any notation then we could use that, otherwise we should probably ask the scientists how they would like to describe them.